templates/named.conf.erb

```diff
@@ -16,5 +16,15 @@ options  {
 include "<%= scope.lookupvar('::dns::localzonepath') %>";
 <% end -%>
 
+<%- unless scope.lookupvar('::dns::acls').empty? -%>
```

Comment:
Like I said, this line is redundant. If acls is empty, the loop won't iterate and no block is shown. You can remove it and the 'with no custom parameters' test still passes. It contains no acl block.